install/kubernetes/helm/meshery/README.md

```diff
@@ -16,10 +16,10 @@ Meshery chart for deploying Meshery
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
 | annotations | object | `{}` |  |
-| env.ADAPTER_URLS | string | `"meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-kuma:10007 meshery-osm:10009 meshery-nginx-sm:10010 meshery-nsm:10004 meshery-app-mesh:10005 meshery-traefik-mesh:10006 meshery-cilium:10012"` |  |
+| env.ADAPTER_URLS | string | `"meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-kuma:10007 meshery-nginx-sm:10010 meshery-nsm:10004 meshery-app-mesh:10005 meshery-traefik-mesh:10006 meshery-cilium:10012"` | Optional, Meshery Server adapters with which to pre-configure Meshery Server. |
```

Comment:
@MUzairS15, I question whether this should even be around anymore.

Reply:
@theBeginner86 will you confirm as well? If not required remove in a subsequent PR, please.